README.md

```diff
@@ -1,15 +1,24 @@
 # slc -- StrongLoop Controller
 
-`slc` is a command-line tool for development and control of your Node
-application.
+`slc` is the command shell for the StrongLoop Process Manager,
```

Comment:
How does `slc arc` fit in to this? I guess it's just the GUI for both parts of `slc`?